src/calculateRentalCost.js

```diff
@@ -29,6 +29,19 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const cost = days * 40;
+
+  if (days < 3) {
+    return cost;
+  } else {
```

Comment:
```suggestion
  }
  
  if (days < 7) {
    // return with small discount
  }
  
  // return with big discount
```
Можно не усложнять структуру, а просто отдельными if определять что возвращать из функции, так как внутри if уже есть return